.github/workflows/run-cluster.yaml

```diff
@@ -3,10 +3,6 @@ name: run-cluster
 on:
   workflow_dispatch:
     inputs:
-      daft_wheel_url:
-        description: Daft python-wheel URL
-        type: string
-        required: false
       daft_version:
         description: Daft version (errors if both this and "Daft python-wheel URL" are provided)
```

Comment:
(Proposal)

This can be changed to allow passing in one of:

1. Daft published version (maybe enforce that it starts with a `v`)
2. Branch/Git SHA (then we have to build)

Reply:
Scoped down to just either running with the:
1. current SHA (using `${{ github.sha }}`)
2. daft-version (by specifying using the input parameter)